libs/shared/assets/src/lib/styles/main.scss

```diff
@@ -12,6 +12,7 @@ body {
 
 html {
   font-size: 16px;
+  scroll-behavior: smooth;
```

Comment:
bad UX - if someone has this turned off in the browser on purpose, we force them to use it.

Reply:
Do you mean to wrap it like this ?
```
@media (prefers-reduced-motion: no-preference) {
  html {
    scroll-behavior: smooth;
  }
}
```